image.go

```diff
@@ -944,6 +944,158 @@ func imageSeenOrWait(ctx context.Context, opt *imageOpt, tag string, dig digest.
 	}
 }
 
+// ImageExportS exports an image to an output stream.
+// The format is compatible with "docker load" if a single image is selected and not a manifest list.
+// The ref must include a tag for exporting to docker (defaults to latest), and may also include a digest.
+// The export is also formatted according to [OCI Layout] which supports multi-platform images.
+// A tar file will be sent to outStream.
+//
+// Resulting filesystem:
+//   - oci-layout: created at top level, can be done at the start
+//   - index.json: created at top level, single descriptor with org.opencontainers.image.ref.name annotation pointing to the tag
+//   - manifest.json: created at top level, based on every layer added, only works for a single arch image
+//   - blobs/$algo/$hash: each content addressable object (manifest, config, or layer), created recursively
+//
+// [OCI Layout]: https://github.com/opencontainers/image-spec/blob/master/image-layout.md
+func (rc *RegClient) ImageExportS(ctx context.Context, rs []ref.Ref, platformType string, outStream io.Writer, opts ...ImageOpts) (err error) {
```

Comment:
We shouldn't duplicate the method for this, it doubles the maintenance and makes every change error prone if it forgets to change the copy. Make it one method with the second method a stub that calls the first, perhaps even deprecated.

Reply:
Have merged！